runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -2554,6 +2555,10 @@ remoteCompile(
          throw;
          }
       }
+   else if (statusCode == compilationVersionCheckFailed)
```

Comment:
When `enableJITaaSPerCompConn` is false, the client stream will be destroyed. How will we do local compilations and how we will avoid future connection requests?